src/nwm_routing/src/nwm_routing/__main__.py

```diff
@@ -1413,6 +1372,13 @@ async def main_v03_async(argv):
             showtiming,
             verbose,
             debuglevel,
+            run_results,
+            data_assimilation_parameters.get("wrf_hydro_lastobs_file",False),
```

Comment:
same comments as above. we can be a bit more parsimonious with the argument list, here.